app/javascript/components/DownloadButton.js

```diff
@@ -108,17 +110,35 @@ function DownloadCommandContainer(props) {
  */
 export default function DownloadButton(props) {
 
+  const searchContext = useStudySearchContext()
+  const userContext = useUserContext()
+
+  const matchingAccessions = searchContext.results.matchingAccessions || [];
+
+  /**
+   * Reports whether Download button be active,
+   * i.e. user is signed in and has search results
+   */
```

Comment:
I don't think this needs to be a function -- if userContext or searchContext changes, the component will rerender anyway.  So this can just be `active = userContext.accessToken !== '' && matchingAccessions.length > 0;`

Reply:
Addressed in b72b604.